triangle.json

```diff
@@ -0,0 +1,98 @@
+        "or instead to signal an error/exception/etc. on an illegal triangle.",
+
+        "If appropriate for your track, you'll need to ensure that no pair of expected values are equal.",
+        "Otherwise, an implementation that always returns a constant value may falsely pass the tests.",
+        "See https://github.com/exercism/xgo/pull/208"
+    ],
+    "cases": [
+        {
+            "description": "equilateral triangle has all sides equal",
+            "sides": [2, 2, 2],
+            "expected": "equilateral"
+        },
+        {
+            "description": "larger equilateral triangle",
+            "sides": [10, 10, 10],
+            "expected": "equilateral"
+        },
+        {
+            "description": "isosceles triangle with last two sides equal",
+            "sides": [3, 4, 4],
+            "expected": "isosceles"
+        },
+        {
+            "description": "isosceles triangle with first two sides equal",
+            "sides": [4, 4, 3],
+            "expected": "isosceles"
+        },
+        {
+            "description": "isosceles triangle with first and last sides equal",
+            "sides": [4, 3, 4],
+            "expected": "isosceles"
+        },
+        {
+            "description": "isosceles triangle with unequal side larger than equal sides",
+            "sides": [4, 7, 4],
+            "expected": "isosceles"
+        },
+        {
+            "description": "scalene triangle has no equal sides",
+            "sides": [3, 4, 5],
+            "expected": "scalene"
+        },
+        {
+            "description": "larger scalene triangle",
+            "sides": [10, 11, 12],
+            "expected": "scalene"
+        },
+        {
+            "description": "scalene triangle with sides in descending order",
+            "sides": [5, 4, 2],
+            "expected": "scalene"
+        },
+        {
+            "#": "Your track may choose to skip this test and deal only with integers if appropriate",
+            "description": "small scalene triangle with floating point values",
+            "sides": [0.4, 0.6, 0.3],
+            "expected": "scalene"
+        },
+        {
+            "description": "a triangle violating the triangle inequality is illegal",
+            "sides": [7, 3, 2],
+            "expected": "illegal"
+        },
+        {
+            "description": "two sides equal, but still violates triangle inequality",
+            "sides": [1, 1, 3],
+            "expected": "illegal"
+        },
+        {
+            "description": "zero is an illegal side length",
+            "sides": [10, 10, 0],
+            "expected": "illegal"
+        },
+        {
+            "description": "triangles with all sides zero are illegal",
+            "sides": [0, 0, 0],
+            "expected": "illegal"
+        },
+        {
+            "description": "negative side lengths are illegal",
+            "sides": [3, 4, -5],
+            "expected": "illegal"
+        }
```

Comment:
> I think one test that given any given valid (negative) number in any position would suffice. 

*If* we decide that we should test negative values, I would argue for at least adding the three cases I describe. We wouldn't want someone to consider (2, -2, -2) an isosceles triangle rather than equilateral, after all.

> I say, then, match it to the readme

And now I have to awkwardly announce that it is not noted in the README. We can fix that, of course!!!